src/test/resources/suite.epam.tc.hw2/HomePageTwoTestSuite

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+<suite name="Home Page Two Test Suite">
+  <test name="Home Page Two">
+    <classes>
+      <class name="com.epam.tc.hw2.ex2.HomePageTwoTest"/>
+    </classes>
+  </test>
+</suite>
```

Comment:
Add a new blank line. Did you run your project via "mvn test"?

Reply:
The blank line is added. Yes I did. The test is running successfully.